documentation/gapc on windows.txt

```diff
@@ -0,0 +1,26 @@
+How to compile gapc on Windows 10:
+
+1) install Linux Sub-System:
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) download Ubuntu app (Ubuntu 18.04 LTS) from Windows-Store
+	1.5) initialize installation by setting Username and Password
+2) prepare compiling and installation of gapc by installing required sources: 
+	2.1) open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
+			       sudo apt install g++ (you need to install both)	
+	2.3) install flex    : sudo apt-get install flex
+	2.4) install bison   : DO NOT: sudo apt-get install bison !! 
+			       INSTEAD visit the bison website and install the latest version
+			       http://mirrors.ibiblio.org/gnu/ftp/gnu/bison/	
+	2.5) install boost   : sudo apt-get install libboost-all-dev (takes some time)
+	2.6) install GSL     : sudo apt-get install libgsl-dev
+	
+	--see also https://github.com/jlab/gapc/blob/master/.travis.yml

```

Comment:
can we add something like "An automated build script to compile Bellman's GAP is used to execute continuous integration tests and might be helpful as further reference: https://github.com/jlab/gapc/blob/master/.travis.yml"